module-system/module-implementations/examples/sov-accessory-state/src/lib.rs

```diff
@@ -0,0 +1,76 @@
+#[cfg(feature = "native")]
```

Comment:
let's add 
```
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```

Reply:
On it

Reply:
Fixed by https://github.com/Sovereign-Labs/sovereign-sdk/pull/773/commits/419b2c1bd1f1c9ea48f0e82984784ab60648b40e.